pkgs/development/python-modules/radon/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+# Python deps
+, mando
+, colorama
+, pytest-mock
+}:
+
+buildPythonPackage rec {
+  pname = "radon";
+  version = "6.0.1";
+
+  src = fetchFromGitHub {
+    owner = "rubik";
+    repo = "radon";
+    rev = "v${version}";
+    hash = "sha256-yY+j9kuX0ou/uDoVI/Qfqsmq0vNHv735k+vRl22LwwY=";
+  };
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-mock
+  ];
+
+  propagatedBuildInputs = [
+    mando
+    colorama
+  ];
+
```

```suggestion
  passthru.optional-dependencies = {
    toml = [
      tomli
    ];
  };
```
https://github.com/rubik/radon/blob/3b75f7905337a02f526ee2deb356267fc4e9f9fb/setup.py#L29-L31